### DISCUSSION TITLE
How can I place the Y axis tick label at the center for horizontal bar chart?

### DISCUSSION THREAD

**Discussion by RaenonX** [recharts/recharts#3473] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/18443743/226793606-eda4c10f-9676-4366-aa88-cc70f27420bd.png)

Ultimately I would like to have the axis ticks places at these "x" places.

**Comment by Yilun-Sun** [comment, marked as answer]
You can simply add a `tickMargin` to the vertical Axis that you have
![image](https://user-images.githubusercontent.com/43896664/228515436-d2277535-70f7-4c4e-96cf-84acca87cfb8.png)

If you want to keep it centered, you may need to calculate the margin by the half of your chart width.
